lib/docs/filters/pig/entries.rb

```diff
@@ -0,0 +1,33 @@
+module Docs
+  class Pig
+    class EntriesFilter < Docs::EntriesFilter
+
+      def include_default_entry?
+        false
```

Comment:
If the page has `additional_entries`, we should never return `false` here. Otherwise the URL without `#fragment` will 404.

I.e., if we have any entry for `/path#id`, we always need one for `/path`. I made that mistake many times in the past and it leads to broken links (within the doc, links are more likely to point to `/path` than `/path#id`).